deployment_scripts/deploy_linux.sh

```diff
@@ -235,7 +239,7 @@ pushd "$ISLAND_PATH/cc/ui" || handle_error
 npm ci
 
 log_message "Generating front end"
-npm run dev
+npm run build
```

Comment:
I'm not sure what value the `dev` build had, but on the new UI we can build production or run a dev server with hot reloads. This builds production in case the dev is not intending to work on UI.

Reply:
Let me rephrase. This is generating the frontend for the old UI (`"$ISLAND_PATH/cc/ui"`). Does `build` work for it?